### DISCUSSION TITLE
AI創作，法律如何規範？

### DISCUSSION THREAD

**Discussion by juliaouo** [1101-GS4719-C/Python-Community-GS4719-C#1492] [General]
近期繪圖 AI 越來越進步，已經到了完整度非常高的程度，不過 AI 算圖有非常多的版權爭議，對於這件事有幾個觀點：
首先是不經過創作者同意便將他的們的創作作為訓練資料去訓練AI，我認為這是一件非常嚴重的事，不過現今的法律還不完善，無法顧及這點。另外創作歸屬問題—當由AI生成的圖像被視為一種創作時，就會引發關於版權歸屬的問題。通常情況下，版權歸屬於創作者，但在AI生成的情況下，創作者可能是算法或AI系統本身，而非人類。這對於法律和藝術界來說都是一個挑戰，因為現行的版權法律通常僅適用於人類創作。再來是人類參與程度—AI算圖過程中人類參與的程度。如果有人類藝術家在AI算法生成的圖像上進行了一定程度的修改或後期處理，是否應該給予他們版權保護？接著是公共領域—有人主張AI生成的圖像應該屬於公共領域，因為它們是由非人類創作者生成的。這一觀點認為，這些圖像應該能夠自由使用，並不受傳統版權限制。
所以政府應該要制訂新的版權法律，以適應新的創作形式，並提供創作者保障。我們需要對法律框架進行反思和更新，以確保公平的版權保護，同時也不阻礙AI技術的發展和創新。

**Comment by pudding0803** [comment]
我目前的想法是應該要公開 AI 的訓練集，或許在訓練集未經他人同意而使用到他人作品時，就要算是牽扯到著作權的問題？否則要判斷是否為 AI 可能在未來會變得極為困難。我目前還想不到其他可以保障創作者的方式或規範，畢竟誰想要什麼時候訓練、亦或是使用 AI 產生創作，應該都難以規範。

**Comment by Elaine2899** [comment]
原本我們的法律就已經跟不上科技許多了，而隨著AI崛起，新的犯罪型態層出不窮，我們的法律到底要怎麼跟上新時代的浪潮，這是一個非常困難的議題。我想，我們需要一個部門，招攬許多具有資訊、法律相關知識背景的人，請他們追著新的浪潮擬定新的規範。或許讓民眾發起新的規範，並確認可行度再加以規範。

**Comment by ashleyy-huang** [comment]
我認為法律本來就應該要跟上時代，當世界上有越多打破常理的東西推出時，社會和政府就應該思考它的倫理及道德性，否則像是AI的繪畫技術就又可能會使的原本的創作者失去了創作的意義且輕易的就被取代了

**Comment by PaulSu0905** [comment]
我認為人工智能生成的藝術引起了版權問題。法律需要更新，以保護創作者，並解決諸如未經同意使用其作品、確定所有權和人類參與程度等問題。公共領域的爭論出現了，但新的法律應該平衡保護和創新。

**Comment by larry40305** [comment]
政府、法律專家和科技界應該攜手合作，進行深入的討論、研究和制定相應的法律和政策。這將有助於確保公平的版權保護，同時促進AI技術的發展和創新，為創作者和社會帶來更大的價值。

**Comment by ThanatosJun** [comment]
聽樓主說「接著是公共領域—有人主張AI生成的圖像應該屬於公共領域」這件事，我認為 生成式 AI 的產出不應該歸類於公共領域底下，除非AI 詠唱者 同意，以圖像生成為例，除了模型外，不同的提示詞與參數調整都會影響產圖的品質與精細程度，可以說在詠唱師的調整下該圖才能藉由 AI 產出精美的圖像。另外說道關於模型訓練的問題，許多手繪創作者的產出都被拿去當作訓練參數，關於這點，也有人提出產出獲利的金額，被當作訓練集的手繪創作者有權拿取，其實很還很多問題，多到可以直接開一門課了！但這篇先這樣！

**Comment by Alex11392** [comment]
我也同意需要新的法律去規範AI領域的著作權，畢竟是新的領域和應用，新的法律能更好的限制。
